src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -0,0 +1,98 @@
+import { ChartType, Plugin } from "chart.js";
+import { chartFontColor } from "../../../../helpers/figures/charts/chart_common";
+import { Color } from "../../../../types";
+
+interface ChartShowValuesPluginOptions {
+  showValues: boolean;
+  background?: Color;
+  horizontal?: boolean;
+  callback?: (any) => any;
+}
+
+declare module "chart.js" {
+  interface PluginOptionsByType<TType extends ChartType> {
+    chartShowValuesPlugin?: ChartShowValuesPluginOptions;
+  }
+}
+
+/** This is a chartJS plugin that will draw connector lines between the bars of a Waterfall chart */
+export const chartShowValuesPlugin: Plugin = {
+  id: "chartShowValuesPlugin",
+  afterDatasetsDraw(chart: any, args, options: ChartShowValuesPluginOptions) {
+    if (!options.showValues) {
+      return;
+    }
+    const drawData = chart._metasets?.[0]?.data;
+    if (!drawData) {
+      return;
+    }
+    const ctx = chart.ctx;
+    ctx.save();
+
+    ctx.textAlign = "center";
+    ctx.textBaseline = "middle";
+    ctx.fillStyle = chartFontColor(options.background);
+    ctx.strokeStyle = chartFontColor(ctx.fillStyle);
+
+    chart._metasets.forEach(function (dataset) {
+      switch (dataset.type) {
+        case "pie": {
+          for (let i = 0; i < dataset._parsed.length; i++) {
```

Comment:
The text seems not really centered in pie chart (sounds annoying to do better, so we can leave it like that if you want)
![image](https://github.com/odoo/o-spreadsheet/assets/31045223/c0b9910d-df16-4588-b920-73a7e4284a84)

But it's not working at all for doughnut charts :x

Reply:
Yes, it really is a pain to do better :/ 

Fixed for doughnut chart :)